Sources/SearchableCollectionViewController/SearchBarView.swift

```diff
@@ -1,34 +1,36 @@
 import Foundation
 import UIKit
 
-class SearchBarView: UICollectionReusableView {
-    
-    static let height: CGFloat = 44.0
-    static let kind = String(describing:SearchBarView.self)
-    static let defaultSearchBackgroundColor = UIColor(hex: "00C365")
-    
-    dynamic open var searchBackgroundColor: UIColor? = SearchBarView.defaultSearchBackgroundColor {
-        didSet { searchBar.backgroundColor = searchBackgroundColor }
+open class SearchBarView: UICollectionReusableView {
```

Comment:
If you share what you want to change we can see if it's something that might be needed in other apps as well and then we can include it here.